manifests/controlled_resources_list.yaml

```diff
@@ -0,0 +1,77 @@
+resources:
```

Comment:
Do we need to add all Istio CRs on an empty installtion of Istio on top of Kyma ? I see more CRs on my instance. We must test this with an integration test, otherwise we risk uninstallation to fail quite recently because we base it on a static list of resources. Unit tested is not enough.

Reply:
Discussed this one, for now let's leave only Istio Operator created resources, we can extend it later